_pages/en/how-we-work.html

```diff
@@ -47,7 +47,7 @@ <h3>Provide advice</h3>
 	<p>We work in the open, sharing what we learn to help make a bigger impact. Housed at the Treasury Board Secretariat, we benefit from line of sight across government and close proximity to the policies and standards needed to drive and scale this transformation.</p>
 
 	<h2><a name="partnering"></a>Interested in partnering with us?</h2>
```

Comment:
actually, there already is an anchor here at line 49, so you shouldn't need to add the one below.  Was the anchor not working before?